<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormClassify</class>
 <widget class="QWidget" name="FormClassify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(70, 96, 99);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(51, 83, 68);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QPushButton" name="selClass01Dir">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>50</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>selClass01Dir</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>331</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(30, 62, 42);
color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>331</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 84, 66);
color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selClass02Dir">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>80</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>selClass02Dir</string>
       </property>
      </widget>
      <widget class="QPushButton" name="train">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>190</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>train</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>class1Dir</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>class2Dir</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QPushButton" name="selTestDir">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>70</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>selTestDir</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="test">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>160</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>test</string>
       </property>
      </widget>
      <widget class="QLabel" name="selTestDir_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>selTestDir</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
